src/webgl/p5.Geometry.js

```diff
@@ -75,6 +78,61 @@ p5.Geometry = class Geometry {
     return this; // TODO: is this a constructor?
   }
 
+  /**
+ * Custom bounding box calculation based on the object's vertices.
+ * The bounding box is a rectangular prism that encompasses the entire object.
+ * It is defined by the minimum and maximum coordinates along each axis, as well
+ * as the size and offset of the box.
+ *
+ *
+ * @method calculateBoundingBox
+ * @memberof p5.Geometry.prototype
+ * @return {Object} An object containing the bounding box properties:
+ *
+ *   - `min`: The minimum coordinates of the bounding box as a p5.Vector.
+ *   - `max`: The maximum coordinates of the bounding box as a p5.Vector.
+ *   - `size`: The size of the bounding box as a p5.Vector.
+ *   - `offset`: The offset of the bounding box as a p5.Vector.
+ *
+ */
+
+  calculateBoundingBox() {
+    if (this.boundingBoxCache) {
+      return this.boundingBoxCache; // Return cached result if available
+    }
+
+    let minVertex = new p5.Vector(
+      Number.MAX_VALUE, Number.MAX_VALUE, Number.MAX_VALUE);
+    let maxVertex = new p5.Vector(
+      Number.MIN_VALUE, Number.MIN_VALUE, Number.MIN_VALUE);
+
+    for (let i = 0; i < this.vertices.length; i++) {
+      let vertex = this.vertices[i];
+      minVertex.x = min(minVertex.x, vertex.x);
```

Comment:
Oh I see tests are still failing, `min` and `max` are also p5 globals and won't be defined in instance mode. I think `Math.min` and `Math.max` will fix it though